content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,123 @@
+# data status
+
+Show changes in data tracked by DVC between the last commit, the dvcfiles and
+the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc data status [-h] [-q | -v]
+                       [--granular] [--unchanged]
+                       [--untracked-files [{no,all}]]
+                       [--json]
+```
+
+## Description
+
+Shows changes that have differences between the last commit (`HEAD`) and
+committed data specified in [metafiles](/doc/user-guide/project-structure)
+files, uncommitted changes that have differences between that specified in the
+metafiles and the <abbr>workspace</abbr>, and the changes that are not in the
+cache but is specified in those metafiles.
+
+The first change shows what is already committed to dvc via `dvc add` or
+`dvc commit` or `dvc repro` but the metafiles are not committed to git yet. The
+second change shows the new changes to the data that is not tracked by DVC yet
+or what you could `dvc commit`. The third change shows the files that are
+missing from the cache. Some of those could be pulled from the remote.
+
+Using `--untracked-files`, it'll show the files that are not being tracked by
+DVC or Git.
+
+By default, `dvc data status` shows changes in the level the file/directory is
+being tracked in. With `--granular`, it will show changes in file-level.
+
+The `--unchanged` option will list all of the files that aren't listed in
+committed changes or uncommitted changes.
+
+### Output
+
+The `dvc data status` displays changes in multiple categories such as:
+
+- _Not in cache_
+- _DVC committed changes_
+- _DVC uncommitted changes_
+- _Untracked files_
+- _DVC unchanged files_
+
+The categories will only be displayed when there are changes related to those
+categories.
+
+The _DVC committed changes_ and _DVC committed changes_ shows more detailed
+state the files are in: _added_, _modified_, and _deleted_.
+
+## Options
+
+- `--granular` - Show granular, file-level information for the changes for
+  dvc-tracked directories. By default, DVC only shows the changes at the level
+  where the data is being tracked at.
+
+- `--untracked-files` - Show files that are not being tracked by DVC and Git.
+  This option can also take values: `no` and `all`. With `--untracked-files=no`,
```

Comment:
Q: why do we need `no` as an option?

Reply:
We might make `normal` default in the future, so `no` will be through `--untracked-files=no`.

Reply:
I still don't get it ... so, no stands for normal or for "don't show"? this is quite confusing ...

Reply:
- `no` stands for "don't show".
- `all` shows all untracked files recursively.
- `normal` only shows untracked directories (without recursing) and files.

Reply:
I'll leave it out for now, as `--untracked-files=all` and `--untracked-files` are same.

Reply:
I see, so, `normal` doesn't exist and is not a default for now. It's fine, but it was not clear from the docs at all. May be we can even skip `no` for now in the docs, since it's not needed? And even skip `all` modifier also ...